src/nni_manager/training_service/kubernetes/kubernetesTrainingService.ts

```diff
@@ -368,12 +365,9 @@ abstract class KubernetesTrainingService {
         } catch (error) {
```

Comment:
after the `retryCount` is used up, you still return the uri.

Reply:
fixed.